src/deployments.js

```diff
@@ -78,6 +55,48 @@ async function inactivatePriorDeployments(context, currentDeploymentNodeId) {
   }
 }
 
+async function getPriorDeploymentStatuses(deploymentNodeIds) {
+  const octokitGraphQl = graphql.defaults({
+    headers: {
+      authorization: `token ${context.token}`
+    }
+  });
+
+  const statuses = [];
+  const statusesQuery = `
+          query($deploymentNodeIds: [ID!]!) {
+            deployments: nodes(ids: $deploymentNodeIds) {
+              ... on Deployment {
+                id
+                databaseId
+                statuses(first:1) {
+                  nodes {
+                    description
+                    state
+                    createdAt
+                  }
+                }
+              }
+            }
+          }`;
+
+  const page = 100;
+  const pages = Math.ceil(params.deploymentNodeIds.length / page);
+  const statusRequests = [];
+
+  for (var i = 0; i < pages; i++) {
+    const sliced = params.deploymentNodeIds.slice(i * page, (i + 1) * page);
```

Comment:
I feel kind of silly asking this but will `params.deploymentNodeIds` work?  The arg being sent in is called `deploymentNodeIds`.  I thought maybe there was a `params` keyword that I was unfamiliar with but I couldn't find anything in the docs.  Am I missing something here?

Reply:
Nope. You're right. I wish it weren't so hard to test this one locally.